Poker.AT/App1aaaaa/App.xaml.cs

```diff
@@ -1,100 +0,0 @@
-﻿using System;
```

Comment:
❓ ❓ ❓  what is this mess? 👎 
![image](https://cloud.githubusercontent.com/assets/9889268/26219794/f2e3aac8-3c18-11e7-886a-ce88b9d282f4.png)